### ISSUE TITLE
"Launch using Discrete Graphics Card" Does not work in application menu.

### ISSUE THREAD

**Issue by MrParKel** [pop-os/pop#2203]
**Distribution (run `cat /etc/os-release`):**
 
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**
"Launch using Discrete Graphics Card" Button won't do anything and just quits in app menu, but it works just fine from launcher on Pop 21.10. My unit has Nvidia GPU on it.

**Steps to reproduce (if you know):**

**Expected behavior:**
It has to run apps with discrete gpu like existing app launcher.

**Other Notes:**

**Comment by MrParKel** [comment]
No one has interest to look at this bug?

**Comment by leviport** [comment]
There is interest, but it seems to be working as expected on my hardware. Can you share the make and model of your laptop? Is it in Hybrid graphics mode when you're seeing this happen? If you run `watch nvidia-smi`, do you see the application appear in the process list at all?

**Comment by MrParKel** [comment]
Yeah I am using Lenovo Ideapad Gaming 3 with Ryzen 5 4600H, and GTX 1650. Yep I checked "nvtop" in terminal and it doesn't opened anything. It's working fine in launcher (Super key launcher) but from app menu it just close the menu and will do nothing.
And it's on Hybrid GPU mode.

**Comment by MrParKel** [comment]
Ok now i found out what happened. Personally i disable DE animations to be faster on work, from the gnome-tweaks i disabled that and if the animation be on the button works fine. But if i turn off the animation the button works if i hit enter on the launch button! It just doesn't work with mouse click, please fix this little bug!

**Comment by mhnghfv** [comment]
I have DE animations enabled yet this still doesn't work.

**Comment by leeli0** [comment]
[Graphics Switching (Pop!_OS)](https://support.system76.com/articles/graphics-switch-pop/), this page resolved my problem by switching to NVIDIA Graphics

![image](https://github.com/user-attachments/assets/1070318a-5436-4c1f-b022-cd29d19ee5ae)
